docs/ros-snap.md

```diff
@@ -0,0 +1,354 @@
+# Using Snappy with ROS
+
+The [Robot Operating System][1] (ROS) is an open-source collection of tools and
+libraries meant to aid in the development of complex robotic systems. ROS is
+excellent at what it does, but there are a few things it doesn't do:
+
+- Security: How does one prevent a rogue application from interfering with one's
+  finely-honed ROS ecosystem? Confinement and access control isn't one of the
+  many problems solved by ROS.
+
+- Deployment: How does one deploy an entire ROS project in one step, without
+  worrying about dependencies?
+
+- Updating: How does one allow end-users to update the entire ROS project in one
+  step, while retaining for the possibility of rolling back the change if
+  the update goes sideways?
+
+These are all problems solved by Snappy; the combination of the two is a perfect
+match for consumer robotic systems.
+
+
+## Package your current ROS project as a .snap
+
+Let's assume you already have a ROS project. It can be as simple or as
+complicated as you like, but for this example, our project will be made up of:
```

Comment:
I actually plan on replacing that example with a more realistic one when I have a better solution for roscore.